frontend-new/src/App.vue

```diff
@@ -1,17 +1,74 @@
 <script setup lang="ts">
-import { useHead } from "@vueuse/head";
-import { useSeo } from "~/composables/useSeo";
+import {useHead} from "@vueuse/head";
+import {onMounted} from 'vue'
+import {useSeo} from "~/composables/useSeo";
+import {useThemeStore} from '~/store/theme'
+
 
 const title = "Hangar New Test";
 const description = "IDK WTF am doing";
 
 useHead(useSeo(title, description, useRoute(), null));
+
+const theme = useThemeStore()
+
+onMounted(() => {
+    if (typeof window !== 'undefined') {
```

Comment:
idk how much stuff we wanna have in app.vue, I think it would be cool to pull that theming stuff out into a composable?